nodemon.json

```
@@ -0,0 +1,5 @@
+{
+  "ignore": ["**/*.test.ts", ".git", "node_modules"],
```

I think we can simplify this to: 

```suggestion
  "ignore": ["**/*.test.ts"],
```

Because `.git` and `node_modules` are ignored per default: https://github.com/remy/nodemon#ignoring-files

---

Jup, thanks

---

removed